pytradfri/gateway.py

```diff
@@ -259,74 +288,119 @@ def factory_reset(cls) -> Command:
 class GatewayInfo:
     """This class contains Gateway information."""
 
-    def __init__(self, raw):
+    _model_class: type[GatewayInfoResponse] = GatewayInfoResponse
```

Comment:
We don't need this attribute since we don't inherit from ApiResponse. We can just create the `raw` instance directly from `GatewayInfoResponse` in the init method below.
```suggestion
```